src/chess/chess_test.rs

```diff
@@ -96,4 +103,40 @@ mod tests {
             assert!(pos.get_flip().get_flip() == pos);
         }
     }
+
+    #[test]
+    fn flip_rand() {
+        let seed: u64 = rand::thread_rng().gen();
+        println!("[{}] Using seed {}", stringify!(flip_rand), seed);
+        let mut rand = StdRng::seed_from_u64(seed);
+
+        let games_num = 100;
+        for _ in 0..games_num {
+            let mut player: Box<dyn GamePlayer<ChessGame>> =
+                Box::new(PlayerRand::from_seed(rand.next_u64() ^ 0xe4655449311aee87));

```

Comment:
why the xor?

Reply:
Didnt look at the rand impl, but lets say its something like this:

init(seed) {self.state=seed}
Int next() {
x=self.state
x^=x<<13
...
return self.state=x
}

So if you init a new rand from another, they we will be identical and produce the same values. Probably the chacha alg is better than this, but xor doesn't damage randomness